### PR TITLE
[docs] update terminating gateway docs for trust store path

### PR DESCRIPTION
**Pull request by kschoche**
### Description
Describe why you're making this change, in plain English.

With Consul Helm chart `0.44.0` the default envoy proxy image is no longer based on the alpine base image, the trust store location is changed in this base image and needs to be updated in our docs when specifying the config-entry.

This should help users avoid situations like: https://github.com/hashicorp/consul-k8s/issues/1262

### Testing & Reproduction steps
* In the case of bugs, describe how to replicate
* If any manual tests were done, document the steps and the conditions to replicate
* Call out any important/ relevant unit tests, e2e tests or integration tests you have added or are adding

### Links
Include any links here that might be helpful for people reviewing your PR (Tickets, GH issues, API docs, external benchmarks, tools docs, etc). If there are none, feel free to delete this section.

Please be mindful not to leak any customer or confidential information. HashiCorp employees may want to use our internal URL shortener to obfuscate links.

### PR Checklist

* [ ] updated test coverage
* [x] external facing docs updated
* [x] not a security concern

### CODE REVIEW

File: website/content/docs/k8s/connect/terminating-gateways.mdx [hunk 1]
```diff
@@ -205,13 +205,15 @@ metadata:
...
 ```
 
 </CodeBlockConfig>
 
-~> If TLS is enabled a `caFile` must be provided, it must point to the system trust store of the terminating gateway
-container (`/etc/ssl/cert.pem`).
+~> If TLS is enabled a `caFile` must be provided, it must point to the system trust store of the terminating gateway container (`/etc/ssl/certs/ca-certificates.crt`).
+
+~> If TLS is enabled and you are using Consul Helm chart 0.43 or older, or an Envoy image with an alpine base
+image, use the following location for the ca certificate file `caFile: /etc/ssl/cert.pem`.
```
Comment (trujillo-adam) [inline website/content/docs/k8s/connect/terminating-gateways.mdx:216]:
```suggestion
If TLS is enabled, you must include the `caFile` parameter  that points to the system trust store of the terminating gateway container. By default, the trust store is located in the `/etc/ssl/certs/ca-certificates.crt` directory.

Configure the `caFile` parameter to point to the `/etc/ssl/cert.pem` directory if TLS is enabled and you are using one of the following components:
 * Consul Helm chart 0.43 or older 
 * Or an Envoy image with an alpine base image
```
I don't think these need to be in callout boxes. They seem to be directly related to the topic being described. 
I also thought the second paragraph might be easier to parse as a list, but feel free to ignore that suggestion.

**Review by trujillo-adam — COMMENTED** [review]
A few suggestions

File: website/content/docs/k8s/connect/terminating-gateways.mdx (hunk 1 above)
Comment (david-yu) [reply website/content/docs/k8s/connect/terminating-gateways.mdx:216]: I agree would be good to call this out in a regular paragraph instead of a callout box.

**Review by david-yu — APPROVED** [review]
